File: Leetcode/Daily/07-2025/C++/01.cpp
// Problem: 3330. Find the Original Typed String I (Easy)
// Link: https://leetcode.com/problems/find-the-original-typed-string-i

class Solution {
public:
    int possibleStringCount(string word) {
        int ans = 1;
        int n = word.size();

        for (int i = 1; i < n; i++) {
            if (word[i] == word[i-1]) ans++;
        }

        return ans;        
    }
};

File: Leetcode/Daily/07-2025/Python3/01.py
# Problem: 3330. Find the Original Typed String I (Easy)
# Link: https://leetcode.com/problems/find-the-original-typed-string-i

class Solution:
    def possibleStringCount(self, word: str) -> int:
        ans = 1
        n = len(word)

        for i in range (1, n):
            if word[i] == word[i-1]:
                ans += 1
        
        return ans